src/dos/dos_programs.cpp

```diff
@@ -1257,12 +1257,13 @@ class IMGMOUNT : public Program {
 				WriteOut_NoParsing(MSG_Get("PROGRAM_IMGMOUNT_SPECIFY_DRIVE"));
 				return;
 			}
-			int i_drive = toupper(temp_line[0]);
-			if (!isalpha(i_drive) || (i_drive - 'A') >= DOS_DRIVES || (i_drive - 'A') <0) {
+			const int i_drive = toupper(temp_line[0]);
+			if (!isalpha(i_drive) || (i_drive - 'A') >= DOS_DRIVES ||
+			    (i_drive - 'A') < 0) {
```

Comment:
Please confirm, but I think this could be rewritten simply as:
```suggestion
			if (!isalpha(i_drive)) {
```

Reply:
I had originally done the same in an earlier push,  but then read how isalpha() takes locale into consideration, so its definition of 'alpha' is expanded beyond the A-Z with locale=C, and we could get other language characters in the mix; so I reverted to leaving it in.

Reply:
This nails it without the subtraction or isalpha:
(i_drive is already upper-cased)
``` c++
	if (i_drive < 'A' || i_drive > 'Z') {
		goto showusage;
	}
```